lib/src/layer.dart

```diff
@@ -55,23 +56,31 @@ class Layer {
   // TMX data format documented here: https://github.com/bjorn/tiled/wiki/TMX-Map-Format#data
   assembleTileMatrix(var bytes) {
     tileMatrix = new List<List<int>>(height);
+    tileRotation = new List<List<List<bool>>>(height);
 
     var tileIndex = 0;
     for (var y = 0; y < height; ++y) {
       tileMatrix[y] = new List<int>(width);
+      tileRotation[y] = new List<List<bool>>(width);
       for (var x = 0; x < width; ++x) {
         var globalTileId =
             bytes[tileIndex] | bytes[tileIndex + 1] << 8 | bytes[tileIndex + 2] << 16 | bytes[tileIndex + 3] << 24;
 
         tileIndex += 4;
 
         // Read out the flags
-        var flipped_horizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG);
-        var flipped_vertically = (globalTileId & FLIPPED_VERTICALLY_FLAG);
-        var flipped_diagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG);
+        bool flippedHorizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG) == FLIPPED_HORIZONTALLY_FLAG;
+        bool flippedVertically = (globalTileId & FLIPPED_VERTICALLY_FLAG) == FLIPPED_VERTICALLY_FLAG;
+        bool flippedDiagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG) == FLIPPED_DIAGONALLY_FLAG;
 
-        // Clear the flags
+        // Save rotation flags
+        tileRotation[y][x] = [
```

Comment:
haven't we decided not to use a list of booleans to store that? 🤔

Reply:
I was working on that! the first commit was just rebasing off current master